doc/reference/aliases.rst

```diff
@@ -0,0 +1,160 @@
+Aliases
+=======
+
+:term:`Aliases <alias>` are build targets that do not correspond to specific
+files. For example, the ``runtest`` alias corresponds to running tests.
+
+Model and Syntax
+----------------
+
+Rules can be attached to an alias. When this alias is requested to be built,
```

Comment:
That's not completely correct. We can attach dependencies to aliases.

We can also attach aliases to rules, although the way it works under the hood is that we attach the actions rather than the rules.